output_frame: map

lidar:
  center:
    ground_removal:
      min_x: -60.0
      max_x: 60.0
      min_y: -60.0
      max_y: 60.0
      min_z: -2.5
      max_z: 0.1
      cell_size: 0.6
      tolerance: 0.15
      filter: average                    # none, median, average, minimum
      filter_size: 3
  voxel_grid_filter:
    leaf_size: 0.25
    filter_field_name: z
    filter_limit_min: -2.0
    filter_limit_max: 0.5
    filter_limit_negative: False
  points_clusterer:
    cluster_epsilon: 0.7
    cluster_min_size: 4
  cluster_detector:
    min_cluster_size: 4
    enable_pointcloud: False
    enable_convex_hull: True
    transform_timeout: 0.06         # needed for Carla

obstacle_simulation:
  publish_rate: 10

ema_tracker:
  detection_counter_threshold: 2
  missed_counter_threshold: 2
  velocity_gain: 0.1
  acceleration_gain: 0.1
  association_method: iou   # iou, euclidean
  max_euclidean_distance: 2.0